src/plugins/github/loadGraph.js

```diff
@@ -15,11 +15,12 @@ import {RelationalView} from "./relationalView";
 import {type RepoId, repoIdToString} from "./repoId";
 import {Graph} from "../../core/graph";
 import {type GithubToken} from "./token";
+import {type CacheProvider} from "../../backend/cache";
 
 export type Options = {|
   +repoIds: $ReadOnlyArray<RepoId>,
   +token: GithubToken,
-  +cacheDirectory: string,
+  +cache: CacheProvider,
```

Comment:
could we call this argument `cacheProvider` rather than just `cache`? When reading the code in `api/load`, i found it confusing that we passed `dataDirectory` as `cache`-- passing it as `cacheProvider` would be less confusing.

Reply:
I'm using cache as the variable name consistently.
Reason is, I think -Provider seems superfluous in variable names.
```js
// Looks interchangeable
cache.database(...);
cacheProvider.database(...);
```

> i found it confusing that we passed dataDirectory as cache

Instead, would it help if I casted dataDirectory to a CacheProvider instead?
```js
{
  cache: (dataDirectory: CacheProvider),
}
```

Reply:
That works too. Since this is a nit and you're the author, I'm fine with either approach.

Reply:
Cast added.